### DISCUSSION TITLE
Custom NER for other languages.

### DISCUSSION THREAD

**Discussion by Sujith1909** [explosion/spaCy#10949] [Language Support]
I hope you are doing well. I read an article (https://towardsdatascience.com/how-to-fine-tune-bert-transformer-with-spacy-3-6a90bfe57647) in which they have used custom training dataset for English articles. My doubt here is if I want to Train the transformer in other language. is it enough if I just change the Language in config file here ( https://spacy.io/usage/training ), can I give  input  in that language as training data or should I change other things too.

**Comment by polm** [comment]
It is enough to change the language in the config, and [the quickstart](https://spacy.io/usage/training#quickstart) does that if you pick your language from the dropdown. 

Depending on the language there might be other settings you could want to adjust, but they shouldn't be necessary to get started.
